### DISCUSSION TITLE
About LineChart

### DISCUSSION THREAD

**Discussion by dlyltm** [dotnetcore/BootstrapBlazor#2245] [Q&A] [answered]
How to set colors of LineChart's x-axis and y-axis?

**Comment by ArgoZhang** [comment]
@dlyltm, do you mean to set the color of the axis text? @azlis

**Reply by dlyltm** [reply]
Not text, but the coordinate axis itself.

**Comment by ArgoZhang** [comment]
@dlyltm Hi 

![292e21504867196affaaf0492cf44042](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/b7f1217a-0db6-42bc-bac8-164c41324816)

Does it meet your requirements?

or this

![28cc95892bd50b4faf2883381bf8bb37](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/ad092d92-a02b-4ffc-b00b-b015ac76310b)

**Reply by dlyltm** [reply]
Yes,it is. But how to set? Can you give an example?

**Comment by ArgoZhang** [comment, marked as answer]
@dlyltm 

### 1 Upgrade chart package to 7.6.1

### 2 Add the following code 

```
var ds = new ChartDataSource();
ds.Options.XScalesBorderColor = "red";
ds.Options.YScalesBorderColor = "red";

ds.Options.XScalesGridColor = "blue";
ds.Options.XScalesGridTickColor = "blue";
ds.Options.XScalesGridBorderColor = "blue";

ds.Options.YScalesGridColor = "blue";
ds.Options.YScalesGridTickColor = "blue";
ds.Options.YScalesGridBorderColor = "blue";
```

ref: https://github.com/dotnetcore/BootstrapBlazor/pull/2248?wt.mc_id=DT-MVP-5004174

**Reply by dlyltm** [reply]
What does "ds" mean? Is it a reference variable of chart?

**Reply by ArgoZhang** [reply]
```
var ds = new ChartDataSource();
```
